### ISSUE TITLE
[FR] Sprite extruder support

### ISSUE THREAD

**Issue by thethirdthree** [bigtreetech/BIGTREETECH-SKR-mini-E3#694]
Running into an issue with the new Sprite extruder with my BL touch.  The much further forward mount is positioning the sensor off the build plate during the auto level feature.  

Is there a firware in the works for this?

**Comment by dvlopez2010** [comment]
That is a issue with the probe offset. You should be able to adjust that in the settings to get it on the build plate. I used the offset for the s1 configuration when compiling the firmware.

**Comment by thethirdthree** [comment]
I'm a little on the stupid side when it comes to that, could you point me in the right direction?

**Comment by dvlopez2010** [comment]
I need more information on where I lost you? In the settings menu you
should be able to find a probe offset setting that allows you to adjust the
offset. If not you will need to compile firmware to adjust the offset.

**Comment by thethirdthree** [comment]
I've got the z-offset dialed in, I'm having an issue with the y shift of the mount.   Where did you find that setting for the S1 config?

**Comment by dvlopez2010** [comment]
I opened the example configuration folder for marlin and scrolled through
the 3300 lines of code until I found them. You can also search for offset
and it will take you there faster.

**Comment by dvlopez2010** [comment]
Yes. I looked up the default offset I had sent to someone else a while
back:

The probe offset default on the s1 firmware is -37, -39

I did not send the z offset but there is a wizard for the z offset.
